lms/envs/aws.py

```diff
@@ -676,6 +676,12 @@
 
 PROGS_URLS = ENV_TOKENS.get('PROGS_URLS', PROGS_URLS)
 
+# Redirection options
+if FEATURES.get("EDRAAK_ENABLE_AUTH_EXTERNAL_REDIRECT"):
+    EDRAAK_AUTH_REDIRECT_ALLOW_ANY = ENV_TOKENS.get("EDRAAK_AUTH_REDIRECT_ALLOW_ANY", False)
+    EDRAAK_AUTH_REDIRECT_ORIGINS_WHITELIST = ENV_TOKENS.get("EDRAAK_AUTH_REDIRECT_ORIGINS_WHITELIST", tuple())
+    EDRAAK_AUTH_REDIRECT_REGX_ORIGINS = ENV_TOKENS.get("EDRAAK_AUTH_REDIRECT_REGX_ORIGINS", tuple())
```

Comment:
```suggestion
EDRAAK_AUTH_REDIRECT_REGX_ORIGINS = ENV_TOKENS.get("EDRAAK_AUTH_REDIRECT_REGX_ORIGINS", tuple())
```

Reply:
Please de-indent, same goes for the features above.

Reply:
Done